docs/compilers/CSharp/Compiler Breaking Changes - post VS2019.md

```diff
@@ -47,4 +47,5 @@ Each entry should include a short description of the break, followed by either a
     ```
 
     Such code will produce an error in version 16.4.
-    
\ No newline at end of file
+
+6. https://github.com/dotnet/roslyn/issues/37527 The constant folding behavior of the compiler differed depending on your host architecture when converting a floating-point constant to an integral type where that conversion would be a compile-time error if not in an `unchecked` context.  We now yield a zero result for such conversions on all host architectures.
```

Comment:
Should an issue be logged against dotnet/coreclr to see if they want to take a similar fix for their constant folding and potentially for the result handling of the `Math.Round` functions?

CoreCLR already took similar fixes for `NaN` (where x86 and ARM return different NaN by default).

Reply:
Given that the result is undefined, I don't care too much that the result is platform-dependent. However, I encourage you to file an issue if you think that would be a good idea.